lib/features/thread_detail/presentation/extension/thread_detail_next_previous_actions.dart

```diff
@@ -40,8 +39,7 @@ extension ThreadDetailNextPreviousActions on ThreadDetailManager {
 
   void _preparePageWithIndex(int index) {
     currentMobilePageViewIndex.value = index;
-    pageController?.dispose();
-    pageController = PageController(initialPage: index);
+    pageController?.jumpToPage(index);
```

Comment:
If pageController == null then it will not jump to the correct page, We should re-initialize it before jumping